java/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Lambda Java
+
+Layers for running Java applications on AWS Lambda with OpenTelemetry.
+
+## Provided layers
+
+Two types of layers are provided
+
+### Java agent
+
+The [OpenTelemetry Java Agent](https://github.com/open-telemetry/opentelemetry-java-instrumentation)
+is bundled into the base of the layer and can be loaded into a Lambda function by specifying the
+environment variable `JAVA_TOOL_OPTIONS=-javaagent:/opt/opentelemetry-javaagent.jar` in your Lambda
+configuration. The agent will automatically instrument your application for all supported libraries.
+Note, automatic instrumentation has a notable impact on startup time on AWS Lambda and you will
+generally need to use this along with provisioned concurrency and warmup requests to serve production
+requests without causing timeouts on initial requests while it initializes.
+
+### Wrapper
```

Comment:
It may make sense to denote that using wrapper will yield only a single span per lambda invocation (plus additional spans if appropriate lib instrumentation is loaded). Out of what I have seen, this (single span) is kind of confusing for users that are used to autoinstrumentation.

Reply:
I tried to capture that with my "For any other library" comment below. Is it too confusing still? Let me know if there's a way to make it clearer.

Reply:
Out of my previous experience with wrapper users I'd state explicitly "wrapper will create a single span for lambda execution" but it may be just me being extra sensitive about that ;)

Reply:
Let's see how it goes, it's a bit different thanks to the AWS SDK instrumentation here